.github/workflows/label-cherry-pick.yaml

```diff
@@ -0,0 +1,71 @@
+name: Label Cherry Pick
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - closed
+
+jobs:
+  cherry_pick:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target_branch: ['23.lts.1+', '22.lts.1+', '21.lts.1+', '20.lts.1+', '19.lts.1+', '19.android.1+', 'rc_11', 'COBALT_9']
+    if: |
+      github.event.pull_request.merged == true &&
+      github.event.pull_request.merge_commit_sha != null
+    env:
+      ACCESS_TOKEN: ${{ secrets.CHERRY_PICK_TOKEN }}
+      REPOSITORY: ${{ github.repository }}
+      GITHUB_REF: ${{ github.ref }}
+      MERGE_COMMIT_SHA: ${{ github.event.pull_request.merge_commit_sha }}
+    steps:
+      - name: Default RUN_FOR_CURRENT_BRANCH to false
+        run: echo "RUN_FOR_CURRENT_BRANCH=false" >> $GITHUB_ENV
+
+      - name: Set RUN_FOR_CURRENT_BRANCH for closed pull_request
+        if: |
+          github.event.action == 'closed' &&
+          github.base_ref != matrix.target_branch &&
+          contains(github.event.pull_request.labels.*.name, format('cp-{0}', matrix.target_branch))
+        run: echo "RUN_FOR_CURRENT_BRANCH=true" >> $GITHUB_ENV
+
+      - name: Set RUN_FOR_CURRENT_BRANCH for labeled pull_request
+        if: |
+          github.event.action == 'labeled' &&
+          github.base_ref != matrix.target_branch &&
+          github.event.label.name == format('cp-{0}', matrix.target_branch)
+        run: echo "RUN_FOR_CURRENT_BRANCH=true" >> $GITHUB_ENV
+
+      - name: Checkout repository
+        if: env.RUN_FOR_CURRENT_BRANCH == 'true'
+        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
```

Comment:
In the other jobs we use the version for verified actions. Why use the hash here?

Reply:
Let's not pin GitHub owned actions.